<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo20</class>
 <widget class="QDialog" name="dlgDemo20">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar Alumnos, Cursos y Profesores en QComboBox</string>
  </property>
  <widget class="QPushButton" name="btnMensaje">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboAlumno">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboCurso">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboProfesor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlumno">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un Alumno</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCurso">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un Curso</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProfesor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un Profesor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
